biapol_utilities/label/_matching_algorithms.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+def max_similarity(label_image_x, label_image_y, similarity_matrix):
+    """
+    Matches labels in two input label images (label_image_x and label_image_y) 
+    based on the maximal value in the similarity_matrix.
+
+    Parameters
+    ----------
+    label_image_x : ND-array, int
+        labelled input image which serves as reference image
+    label_image_y : ND-array, int
+        Labbelled input image, the labels of which will be matched up with the 
+        labels in label_image_x based on the chosen input metric
+    metric_matrix : 2D-array, float
```

Comment:
```suggestion
    similarity_matrix : 2D-array, float
```
Please name the parameters in the docstring like in the function definition. Find out how your IDE can do this for you automatically.